ltu/engine/client.py

```diff
@@ -23,17 +24,37 @@ def __init__(self, application_key, server_url):
       server_url:       complete http url to the OnDemand server.
     """
     self.application_key = application_key
-    self.server_url      = server_url
-    if not self.server_url[-1] == '/':
-      self.server_url += '/'
+    self.base_server_url = server_url
```

Comment:
This is unclear what is achieved here. It seems to perform the following:
```python
self.server_url = server_url.rstrip('/')
```
Is it ?